docs/usage/configuration.md

```diff
@@ -34,7 +34,7 @@ spec:
       apiVersion: rsyslog-relp.extensions.gardener.cloud/v1alpha1
       kind: RsyslogRelpConfig
       # Set the target server to which logs are sent. The server must support the RELP protocol.
-      target: some.rsyslog-rlep.server
+      target: some.rsyslog-relp.server
```

Comment:
You could also fix similar typo in https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/c8be0cbeba2f36ead53222546f0a9b2e7edde1d8/pkg/cmd/rsyslogrelp/options.go#L5.